tests/integration/basic/test_categorical.py

```diff
@@ -17,7 +17,7 @@ def setup_predictor(self, df, target):
         predictor = predictor_from_problem(df, ProblemDefinition.from_dict(
             {'target': target, 'time_aim': 20, 'unbias_target': False}))
 
-        predictor.learn(train)
+        train_and_check_time_aim(predictor, train)
```

Comment:
do we need to do a time-aim check on ALL unit-tests? If there is an issue with time_aim (for whatever reason) downstream, every unit-test will fail. I don't know if it's helpful to have a condition where we can't isolate which problem is there.